src/olympia/activity/tests/test_models.py

```diff
@@ -99,6 +99,89 @@ def test_rejected_version_still_valid(self):
         assert self.token.is_valid()
 
 
+class TestActivityLogManager(TestCase):
+    def test_pending_for_developer(self):
```

Comment:
FIXME: need more tests, particularly when using the same activitylog for multiple replies across different versions, but also the special `for_version` arg